src/webapp/src/client/app/kudos/wall-widget/wall-widget.html

```diff
@@ -2,16 +2,16 @@
 <!-- <div id="kudos-wall-widget" ng-if="hasKudosPermissions"> -->
     <div class="kudos-big-buttons-wrapper">
         <div class="kudos-big-button">
-            <a ace-kudosify-modal ga-track-event="['Kudos', 'wall big plus', 'opened']" data-test-id="kudosify-button-plus">
+            <a ace-kudosify-modal kudosify-type="{{vm.modalTypes.submit}}" ga-track-event="['Kudos', 'wall big plus', 'opened']" data-test-id="kudosify-button-plus">
                 <span uib-tooltip="{{'kudos.submitKudos' | translate}}" 
                       tooltip-placement="bottom" 
                       tooltip-class="word-break" 
                       class="glyphicon glyphicon-plus kudosify-big-plus-btn" 
                       aria-hidden="true"></span>
             </a>
         </div>
-        <div class="kudos-big-button" feature-flag="kudosSendImproved">
```

Comment:
If "kudosSendImproved" feature-flag no longer needed, then it should be also removed from `config.js`